npm/private/npm_translate_lock_generate.bzl

```diff
@@ -989,6 +990,14 @@ load("@aspect_rules_js//npm/private:npm_package_store.bzl", _npm_package_store =
                         repo_package_json_bzl = repo_package_json_bzl,
                     ),
                 ]))
+                resolved_json_file_path = paths.normalize(paths.join(link_package, _import.package, _RESOLVED_JSON_FILENAME))
```

Comment:
This is currently in the `if _import.package_info.get("has_bin"):` and should either be moved up a level or put into a new loop?